spec/requests/shows_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+load "#{Rails.root}/db/seeds.rb"
```

Comment:
This might be a bit too much for this PR so feel free to ignore this as a suggested change, but just FYI it's generally a good idea to use fixture files of a tool like FactoryBot to generate test data, to keep your specs lean, performant, and more tightly scoped

Reply:
ye I would not use seeds here.

Reply:
also how does this work....I dont not see the code that converts seeds to AR anywhere :o

Reply:
I guess same here 😅 you still did not implement this I guess.